stdlib/Makefile

```diff
@@ -56,6 +56,16 @@ allopt: stdlib.cmxa std_exit.cmx
 opt.opt: allopt
 
 .PHONY: install
+# Ensure any pre-4.13 lowercased artefacts are removed on macOS and Windows
+install::
+	stale="$(filter-out $(notdir $(wildcard stdlib__*.cmi)), \
+         $(notdir $(wildcard $(INSTALL_LIBDIR)/stdlib__*.cmi)))"; \
+  if test -n "$$stale" ; then \
+    echo "$(INSTALL_LIBDIR) contains stale stdlib artefacts"; \
+    echo "Please rm $(INSTALL_LIBDIR)/stdlib__*.cm* and re-run make install"; \
```

Comment:
```suggestion
    echo "Please rm $(INSTALL_LIBDIR)/stdlib__*.cmi and re-run make install"; \
```
?

Reply:
Well noticed 🙂 My theory here was that it would be annoying to be told at this stage to delete the .cmi files only to be told when you rerun `make install` that you should now also delete the .cmx files. The other message can only be reached (officially) if the `.cmi` files had already been deleted.

Reply:
Makes sense, thanks.

Reply:
If cmt and cmti are already present can this cause issue ? The variable stale only check for cmi.

Reply:
You'd need to have installed 4.07-4.12, manually removed only the .cmi files and then also be in the relatively unusual position of wanting to install 4.13+ over the top of that manually-broken old installation 🙂

Source artefacts are installed in the `install::` rule following this one - there could be the same check there, but it feels somewhat overkill to add it?